devices/lixee.js

```diff
@@ -0,0 +1,697 @@
+    exposes.numeric('SINSTS3', ea.STATE).withUnit('VA').withProperty('apparentPowerPhC').withDescription('Immediate apparent power delivered (phase 3)'),
+    exposes.numeric('SMAXN', ea.STATE).withUnit('VA').withProperty('activePowerMax').withDescription('Apparent power delivered peak'),
+    exposes.numeric('SMAXN1', ea.STATE).withUnit('VA').withProperty('activePowerMax').withDescription('Apparent power delivered peak (phase 1)'),
+    exposes.numeric('SMAXN2', ea.STATE).withUnit('VA').withProperty('activePowerMaxPhB').withDescription('Apparent power delivered peak (phase 2)'),
+    exposes.numeric('SMAXN3', ea.STATE).withUnit('VA').withProperty('activePowerMaxPhC').withDescription('Apparent power delivered peak (phase 3)'),
+    exposes.numeric('SMAXN-1', ea.STATE).withUnit('VA').withProperty('drawnVAMaxN1').withDescription('Apparent power max. draw-off n-1'),
+    exposes.numeric('SMAXN1-1', ea.STATE).withUnit('VA').withProperty('drawnVAMaxN1').withDescription('Apparent power max. draw-off n-1 (phase 1)'),
+    exposes.numeric('SMAXN2-1', ea.STATE).withUnit('VA').withProperty('drawnVAMaxN1P2').withDescription('Apparent power max. draw-off n-1 (phase 2)'),
+    exposes.numeric('SMAXN3-1', ea.STATE).withUnit('VA').withProperty('drawnVAMaxN1P3').withDescription('Apparent power max. draw-off n-1 (phase 3)'),
+    exposes.text('MSG1', ea.STATE).withProperty('message1').withDescription('Message short'),
+    exposes.text('MSG2', ea.STATE).withProperty('message2').withDescription('Message ultra-short'),
+    exposes.text('PRM', ea.STATE).withProperty('siteId'),
+    exposes.numeric('DPM1', ea.STATE).withProperty('startMobilePoint1').withDescription('Start mobile point 1'),
+    exposes.numeric('FPM1', ea.STATE).withProperty('stopMobilePoint1').withDescription('Stop mobile point 1'),
+    exposes.numeric('DPM2', ea.STATE).withProperty('startMobilePoint2').withDescription('Start mobile point 2'),
+    exposes.numeric('FPM2', ea.STATE).withProperty('stopMobilePoint2').withDescription('Stop mobile point 2'),
+    exposes.numeric('DPM3', ea.STATE).withProperty('startMobilePoint3').withDescription('Start mobile point 3'),
+    exposes.numeric('FPM3', ea.STATE).withProperty('stopMobilePoint3').withDescription('Stop mobile point 3'),
+    exposes.numeric('RELAIS', ea.STATE).withProperty('relais'),
+    exposes.numeric('NJOURF', ea.STATE).withProperty('daysNumberCurrentCalendar').withDescription('Current day number supplier calendar'),
+    exposes.numeric('NJOURF+1', ea.STATE).withProperty('daysNumberNextCalendar').withDescription('Next day number supplier calendar'),
+    exposes.text('PJOURF+1', ea.STATE).withProperty('daysProfileCurrentCalendar').withDescription('Profile of the next supplier calendar day'),
+    exposes.text('PPOINTE1', ea.STATE).withProperty('daysProfileNextCalendar').withDescription('Profile of the next check-in day'),
```

Comment:
Expose `energy` for Energy module in homeassistant

```suggestion
    exposes.text('PPOINTE1', ea.STATE).withProperty('daysProfileNextCalendar').withDescription('Profile of the next check-in day'),
    exposes.numeric('energy', ea.STATE).withUnit('Wh').withProperty('currentSummDelivered').withDescription('Index Base')
```

Reply:
Here, I'm not sure doing it this way. Basically, `energy` is a restriction of how Z2M was implemented for submitting this metadata to HA. I did a PR in order to solve the issue (https://github.com/Koenkk/zigbee2mqtt/pull/10133), but my Linky is in Historique mode right now, so I was not able to test how is working in Standard mode.

Theorically, just with entities BASE (Historique) and EAST (Standard) should be enough, but maybe it is interesting a universal entity for both cases. What do you think?